utils/types.go

```diff
@@ -24,6 +24,24 @@ type Kubernetes struct {
 	NodeName   string `json:"node_name"`
 }
 
+type Args struct {
+	Mesos Mesos `json:"org.apache.mesos,omitempty"`
+}
+
+type Mesos struct {
+	NetworkInfo NetworkInfo `json:"network_info"`
+}
+
+type NetworkInfo struct {
+	Name string `json:"name"`
+	Labels struct {
```

Comment:
I notice that Tom did this below, but I much prefer not inlining these structs like this.

```go
...
Labels mesosLabelList
...

type mesosLabelList struct {
  Labels []mesosLabel
}

type mesosLabel struct {
  Key string
  Value string
}
```

Is more clear to me - what do you think?